modules/nf-core/truvari/consistency/main.nf

```diff
@@ -0,0 +1,51 @@
+
+process TRUVARI_CONSISTENCY {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/truvari:4.1.0--pyhdfd78af_0':
+        'biocontainers/truvari:4.1.0--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf1), path(vcf2), path(vcf3)
```

Comment:
Can this only be calculated on 2 or 3 VCFs?

Reply:
yes, either 2 or 3 vcfs

Reply:
okay then this is fine :)